From b1ad0a4560ff3fecc45bf54c64688ffa6a43c714 Mon Sep 17 00:00:00 2001
From: Yan <yanxk.fnst@fujitsu.com>
Date: Fri, 6 Oct 2023 14:03:50 +0800
Subject: [PATCH] modify makefile for cross build

remove static cc ar ranlib
add cflag and ldflag to build command

Upstream Status: Pending
Signed-off-by: Yan <yanxk.fnst@fujitsu.com>
---
 Makefile | 9 +++------
 1 file changed, 3 insertions(+), 6 deletions(-)

diff --git a/Makefile b/Makefile
index 8be64c3..4659342 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,4 @@
 CFLAGS=-g -Wall
-CC=gcc
-AR=ar
-RANLIB=ranlib
 LIBS=-L./ -lcgic
 
 all: libcgic.a cgictest.cgi capture
@@ -19,14 +16,14 @@ libcgic.a: cgic.o cgic.h
 #mingw32 and cygwin users: replace .cgi with .exe
 
 cgictest.cgi: cgictest.o libcgic.a
-	gcc cgictest.o -o cgictest.cgi ${LIBS}
+	$(CC) $(CFLAGS) $(LDFLAGS) cgictest.o -o cgictest.cgi ${LIBS}
 
 capture: capture.o libcgic.a
-	gcc capture.o -o capture ${LIBS}
+	$(CC) $(CFLAGS) $(LDFLAGS) capture.o -o capture ${LIBS}
 
 clean:
 	rm -f *.o *.a cgictest.cgi capture cgicunittest
 
 test:
-	gcc -D UNIT_TEST=1 cgic.c -o cgicunittest
+	$(CC) -D UNIT_TEST=1 cgic.c -o cgicunittest
 	./cgicunittest
-- 
2.25.1

